docs/tutorial.rst

```diff
@@ -121,22 +125,26 @@ There is a preferred way to resolve this. If any kind of fitting object has been
 containing an instance of ``Model``. This can again be called::
 
     model = {y_1: a * x**2, y_2: 2 * x * b}
-    fit = Fit(model, x=xdata)
+    fit = Fit(model, x=xdata, y_1=y_data1, y_2=y_data2)
     fit_result = fit.execute()
 
     y_1, y_2 = fit.model(x=xdata, **fit_result.params)
 
-This returns a tuple with the components evaluated so through the magic of tuple unpacking``y_1`` and ``y_2`` contain the
-evaluated fit. Nice!
+This returns a named tuple with the components evaluated so through the magic of tuple unpackingi ``y_1`` and ``y_2`` contain the
+evaluated fit. Nice! It is usually beter to do the unpacking explicitely though.
```

Comment:
Disagree with that last statement. First line of the style guide:

> Anything Raymond Hettinger says wins the argument until I have time to write a proper style guide.

Joking aside, in simple cases such as this, tuple unpacking is just so much clearer and easier. And alphabetic ordering of the dependent variables is guaranteed so no ambiguity there.